jar-infer/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -169,4 +206,10 @@ public void toy2() {
         "  }",
         "}");
   }
+
+  @Test
+  public void toy3() {
+    testJARTemplate(
+        "toyJAR", "toys", "./src/test/resources/com/uber/nullaway/jarinfer/testdata/toys.jar");
+  }
```

Comment:
Are we checking the output here somehow? Otherwise it is not a very useful test. Might need a minimal StubxReader, or even just a check that the file exists and has a expected hash.